app/views/games/_chessboard.html.erb

```diff
@@ -1,42 +1,61 @@
 <script>
-
- $(function(){
-
-  $(".piece").draggable({
-    update: function(event, ui){
-
-
-    }
-  })
- });
+$(document).ready(function () {
+  $(function() {
```

Comment:
This syntax (`$(function() {`) is functionally identical to the line above it (`$(document).ready(function () {`) - they're both ways to set up a callback for when the page is loaded.